### PR TITLE
change / remove  "web technologies" and HTML/CSS

### PR DESCRIPTION
**Pull request by jasonkneen**
Updated features to remove emphasis on HTML / web technologies, focus on Native.

### CODE REVIEW

**Comment by hansemannn** [comment]
Approved. I talked with @ingo a few months ago that we should refactor the whole readme-file to be more modern. This includes not only fresh content but also a more Axway / Appc styled design. We may want to open a ticket for that. @jasonkneen would you like to handle that? Thx!

**Comment by jasonkneen** [comment]
sure thing

File: README.md [hunk 1]
```diff
@@ -23,11 +23,11 @@ With Titanium, you use JavaScript to code your application.  Titanium's compiler
...
 Titanium writes native code so you don't have to. :)
 
-- Native apps built using web technologies
-- Apps are compiled and run locally (offline available)
-- Full support for HTML5 and CSS3
-- Support for all native platform UI controls
+- Native apps built using JavaScript
+- Apps are compiled and run locally with full offline support.
+- Support for native platform UI controls
 - Third-party JavaScript support such as jQuery, Dojo, etc.
```
Comment (jhaynie) [inline README.md]: Why not just say instead `"Native Platform UI Controls (vs. Web styled controls to look like Native)` or something more specific about what you mean?

File: README.md [hunk 2]
```diff
@@ -23,11 +23,11 @@ With Titanium, you use JavaScript to code your application.  Titanium's compiler
...
+- Support for Web views incoporating HTML5, CSS etc.
```
Comment (jhaynie) [inline README.md]: `incoporating` -> `incorporating`

**Comment by hansemannn** [comment]
Agreeing on that. We also currently don't mention Hyperloop in this repo, although it's clearly worth to mention. Many developers rely on the github representation of OSS-projects, so we should refresh it sooner than later. 🚀

**Pull request merged**